File: credit/src/main/java/ru/patterns/credit/shared/response/credit/create/Account.java
package ru.patterns.credit.shared.response.credit.create;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

public record Account(
        UUID id,
        LocalDateTime creationTimestamp,
        LocalDateTime blockedTimestamp,
        LocalDateTime closedTimestamp,
        UUID clientId,
        String number,
        BigDecimal balance,
        boolean isCredit
) {
}

File: mobile_app/app/src/main/java/com/example/patterns/DataLayer/Entity/Account.kt
package com.example.patterns.DataLayer.Entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "accounts")
data class Account(
    @PrimaryKey val id: String,
    val creationTimestamp: String,
    val blockedTimestamp: String?,
    val closedTimestamp: String?,
    val clientId: String,
    val number: String,
    val balance: Double,
    val isCredit: Boolean,
    val currency: String
)